include/ofi_list.h

```diff
@@ -200,11 +200,13 @@ static inline void dlist_splice_tail(struct dlist_entry *head,
 struct dlist_ts {
 	struct dlist_entry	*head;
 	fastlock_t		lock;
+	struct dlist_entry	divider;
```

Comment:
I missed this before.  Please remove divider, rename 'head' to 'list', and convert it to dlist_entry, rather than a pointer.

Reply:
Done